pallets/ajuna-awesome-avatars/src/lib.rs

```diff
@@ -1639,5 +1734,36 @@ pub mod pallet {
 			let season = Seasons::<T>::get(season_id).ok_or(Error::<T>::UnknownSeason)?;
 			Ok(season)
 		}
+
+		fn try_propagate_chain_fee(
+			account: &T::AccountId,
+			base_fee: BalanceOf<T>,
+		) -> Result<BalanceOf<T>, DispatchError> {
+			if let Some(mut chain) = T::AffiliateHandler::get_affiliator_chain_for(account) {
+				let mut final_fee = base_fee;
+
+				if let Some(chain_acc) = chain.pop() {
+					let transfer_fee = final_fee
+						.saturating_mul(10_u32.into())
```

Comment:
I thought those rules are from the affiliate pallet

Reply:
when you set the set_global_config extrinsic, we check that the affiliate.level_fee cumulated never passes 100.
[10, 5, 0, 0, 0] for now the array is fine with 5